_posts/en/newsletters/2022-01-05-newsletter.md

```diff
@@ -0,0 +1,209 @@
+  the original implementation of segwit, Bitcoin Core has defaulted to
+  not relaying or mining unconfirmed transactions that spend segwit
+  inputs with uncompressed public keys.  That means any user who creates
+  an address using an uncompressed key may not be able to spend any
+  funds received to that address.  For that reason, these RPCs never
+  create [bech32][topic bech32] addresses for uncompressed keys---they
+  instead create legacy (base58check) addresses.  The new warning field
+  should help users in this situation understand why they're receiving a
+  different address type than what they requested.
+
+- [Bitcoin Core GUI #459][] updates the generate-address dialog with the
+  ability to create bech32m addresses in addition to older address
+  types.  FIXME:harding:add screenshot
+
+- [Eclair #2090][] adds support for rate limiting onion messages via the
+  "max-per-peer-per-second" configuration option.
+
+- [Eclair #2104][] adds a log message that warns the local node operator
+  when their immediately onchain spendable balance has dropped below the
+  estimated amount necessary to use [CPFP fee bumping][topic cpfp] with
+  [anchor outputs][topic anchor outputs] to close their channels in a
+  timely manner.  LN implementators or operators choosing their own
+  reserve values may wish to compare the estimates Eclair has made to
+  [those][news113 lnd4908] used [in][news149 lnd5274] LND.
+
+- [Eclair #2099][] adds an `onion-messages` configuration option that
+  can be set to never relay [onion messages][topic onion messages] (but
+  still allow the node to send or receive messages), relay all messages
+  (even those that require opening a connection to a new peer), or only
+  relay messages over existing connections.
+
+- [Libsecp256k1 #964][] outlines a release process and a versioning scheme for
+  the libsecp256k1 library.
+
+- [Rust Bitcoin #681][] adds support for [BIP371][]'s additional
+  [PSBT][topic psbt] fields for [taproot][topic taproot].
+
+- [Rust-Lightning #1177][] drops the need for Rust-Lightning itself to
+  store information about payments the higher-level wallet application
+  wants to receive.  Instead, the essential information about the
+  payment is encrypted and then encoded into the [payment secret][topic
+  payment secrets].  When a payment is received, the encrypted payment
+  secret is decrypted and the plaintext information is used to derive
+  the [payment preimage][] used to satisfy the [payment hash][] used to
+  secure the payment's [HTLC][topic htlc].
+
+    This is a simplified implementation of the idea described in
+    [Newsletter #168][news168 stateless].  Other LN implementations
+    would likely need an additional field added to [BOLT11][] invoices,
+    as described in [BOLTs #912][], but Rust-Lightning sidesteps that as
+    a library which allows the higher-level application to manage the
+    details of its own payment requests.
```

Comment:
The last half-sentence throws me off a bit. Does Rust-Lightning do less work and therefore not need the field?

Reply:
Significantly revising this to address your concern; let me know (even post publication) if it doesn't quite work.